README.md

```diff
@@ -737,28 +744,43 @@ configuration bucket.
 If you get the following exception:
 ```
 An error occurred (AccessDeniedException) when calling the GetParameter operation: User: arn:aws:sts::{account_id}:assumed-role/azul-indexer-{deployment_stage}/azul-indexer-{deployment_stage}-index is not authorized to perform: ssm:GetParameter on resource: arn:aws:ssm:{aws_region}:{account_id}:parameter/dcp/dss/{deployment_stage}/environment: ClientError
-Traceback (most recent call last):  
+Traceback (most recent call last):
     ...
 botocore.exceptions.ClientError: An error occurred (AccessDeniedException) when calling the GetParameter operation: User: arn:aws:sts::{account_id}:assumed-role/azul-indexer-{deployment_stage}/azul-indexer-{deployment_stage}-index is not authorized to perform: ssm:GetParameter on resource: arn:aws:ssm:{aws_region}:{account_id}:parameter/dcp/dss/integration/environment
 ```
 
-Check whether the DSS switched buckets. If so, the lambda policy may need to be 
-updated to reflect that change. To fix this, redeploy the lambdas (`make 
+Check whether the DSS switched buckets. If so, the lambda policy may need to be
+updated to reflect that change. To fix this, redeploy the lambdas (`make
 package`) in the affected deployment.
 
+## `make requirements_update` does not update transitive requirements
+
+In some cases, `make requirements_update` might not produce any updates to
+transitive requirements, even if you expect them. For example, a sandbox build
+on Gitlab might identify updated transitive requirements even though doing `make
+requirements_update` locally doesn't.
+
+This is a side effect of the Docker build cache on two different machines
+diverging to reflect different states on PyPI. This can be fixed by running
+`make requirements_update_force` instead.
+
+```
+make requirements_update_force
+```
```

Comment:
This is redundant. My previous comment https://github.com/DataBiosphere/azul/pull/2360#discussion_r512863923 included the removal.

Reply:
My oversight.